cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/tests.rs

```diff
@@ -346,3 +348,202 @@ mod bridge_hub_rococo_tests {
 		);
 	}
 }
+
+mod bridge_hub_bulletin_tests {
+	use super::*;
+	use bridge_common_config::BridgeGrandpaRococoBulletinInstance;
+	use bridge_to_bulletin_config::{
+		RococoBulletinChainId, RococoBulletinGlobalConsensusNetwork,
+		WithRococoBulletinMessageBridge, WithRococoBulletinMessagesInstance,
+		XCM_LANE_FOR_ROCOCO_PEOPLE_TO_ROCOCO_BULLETIN,
+	};
+
+	#[test]
+	fn initialize_bridge_by_governance_works() {
+		// for Bulletin finality
+		bridge_hub_test_utils::test_cases::initialize_bridge_by_governance_works::<
+			Runtime,
+			BridgeGrandpaRococoBulletinInstance,
+		>(
+			collator_session_keys(),
+			bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+			Box::new(|call| RuntimeCall::BridgePolkadotBulletinGrandpa(call).encode()),
+		)
+	}
+
+	#[test]
+	fn handle_export_message_from_system_parachain_add_to_outbound_queue_works() {
+		// for Bulletin
+		bridge_hub_test_utils::test_cases::handle_export_message_from_system_parachain_to_outbound_queue_works::<
+			Runtime,
+			XcmConfig,
+			WithRococoBulletinMessagesInstance,
+		>(
+			collator_session_keys(),
+			bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+			SIBLING_PARACHAIN_ID,
+			Box::new(|runtime_event_encoded: Vec<u8>| {
+				match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
+					Ok(RuntimeEvent::BridgePolkadotBulletinMessages(event)) => Some(event),
+					_ => None,
+				}
+			}),
+			|| ExportMessage {
+				network: RococoBulletinGlobalConsensusNetwork::get(),
+				destination: X1(Parachain(1234)),
+				xcm: Xcm(vec![]),
+			},
+			XCM_LANE_FOR_ROCOCO_PEOPLE_TO_ROCOCO_BULLETIN,
+			Some((TokenLocation::get(), ExistentialDeposit::get()).into()),
+			// value should be >= than value generated by `can_calculate_weight_for_paid_export_message_with_reserve_transfer`
+			Some((TokenLocation::get(), bp_bridge_hub_rococo::BridgeHubRococoBaseXcmFeeInRocs::get()).into()),
+			|| (),
+		)
+	}
+
+	#[test]
+	fn message_dispatch_routing_works() {
+		// from Bulletin
+		bridge_hub_test_utils::test_cases::message_dispatch_routing_works::<
+			Runtime,
+			AllPalletsWithoutSystem,
+			XcmConfig,
+			ParachainSystem,
+			WithRococoBulletinMessagesInstance,
+			RelayNetwork,
+			RococoBulletinGlobalConsensusNetwork,
+		>(
+			collator_session_keys(),
+			bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+			SIBLING_PARACHAIN_ID,
+			Box::new(|runtime_event_encoded: Vec<u8>| {
+				match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
+					Ok(RuntimeEvent::ParachainSystem(event)) => Some(event),
+					_ => None,
+				}
+			}),
+			Box::new(|runtime_event_encoded: Vec<u8>| {
+				match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
+					Ok(RuntimeEvent::XcmpQueue(event)) => Some(event),
+					_ => None,
+				}
+			}),
+			XCM_LANE_FOR_ROCOCO_PEOPLE_TO_ROCOCO_BULLETIN,
+			|| (),
+		)
+	}
+	/*
```

Comment:
can we uncomment this now?

Reply:
I've been pretty sure that I did that already :confused:  Lost that commit somewhere, so did it again from the scratch :)